oauth.coffee

```diff
@@ -64,6 +64,12 @@ class OAuth2Server
 
 			if client.redirectUri isnt req.query.redirect_uri
 				return res.redirect '/oauth/error/invalid_redirect_uri'
+			if Array.isArray(client.redirectUri)
```

Comment:
what about change this if/else to something like:
```coffeescript
if not [].concat(client.redirectUri).includes(req.query.redirect_uri)
	return res.redirect '/oauth/error/invalid_redirect_uri'
```

using `[].concat` it doesn't matter if the value is already an array or not =)